src/controller/java/BUILD.gn

```diff
@@ -122,12 +114,28 @@ android_library("java") {
     "zap-generated/chip/devicecontroller/ClusterWriteMapping.java",
   ]
 
+  if (build_java_matter_controller) {
```

Comment:
It seems this is not yet defined, maybe this is some dependency chain? We could add the flag in this PR as well with some TODO.

Reply:
this is accidental insertion from my another PR, just remove the change from this PR.